pkg/runtime/wfengine/component.go

```diff
@@ -95,22 +95,19 @@ func (c *workflowEngineComponent) Start(ctx context.Context, req *workflows.Star
 	if req.Options != nil {
 		if startTimeRFC3339, ok := req.Options["dapr.workflow.start_time"]; ok {
 			if startTime, err := time.Parse(time.RFC3339, startTimeRFC3339); err != nil {
-				return nil, fmt.Errorf("start times must be in RFC3339 format (e.g. \"2009-11-10T23:00:00Z\")")
+				return nil, errors.New(`start times must be in RFC3339 format (e.g. "2009-11-10T23:00:00Z")`)
 			} else {
 				opts = append(opts, api.WithStartTime(startTime))
 			}
 		}
 	}
 
-	var workflowID api.InstanceID
-	var err error
-
-	workflowID, err = c.client.ScheduleNewOrchestration(ctx, req.WorkflowName, opts...)
+	workflowID, err := c.client.ScheduleNewOrchestration(ctx, req.WorkflowName, opts...)
 	if err != nil {
 		return nil, fmt.Errorf("unable to start workflow: %w", err)
 	}
 
-	c.logger.Infof("created new workflow instance with ID '%s'", workflowID)
+	c.logger.Debugf("Created new workflow instance with ID '%s'", workflowID)
```

Comment:
Why are you changing all the statements for starting, stopping, raising event, etc.. from info to debug?

Reply:
We normally treat these kinds of logs as debug logs in APIs, since they can add a lot of noise for heavy workflow users. But I can revert this if needed.

Reply:
I'd love to better understand how we think about logging in Dapr more generally. I tried to be really selective about which logs were promoted to Info, figuring that starting a workflow may be something that an operator looking through production logs might need to know about. But happy to learn if we think there are better ways to help provide these kinds of operational insights (OTel traces?).

Reply:
I'm fine with the logs either way, but I think in this case debug does make sense as you could get a very similar log to this with API logging, right? If you're just looking for the calls, that's what I would do. If I have was having issues with my workflow, that's when I'd want this log and would turn on debug logging to see the extra info.